rcp/org.testeditor.rcp4.uatests/pom.xml

```diff
@@ -229,6 +229,7 @@
 				<groupId>org.eclipse.tycho</groupId>
 				<artifactId>tycho-surefire-plugin</artifactId>
 				<configuration>
+					<argLine>-Dorg.testeditor.ignoreConnectionStare=true</argLine>
```

Comment:
The `argLine` should not be set explicitly here (this will intefere with jacoco and crash the tests on OS X).  Please overwrite the variable `tycho-surefire-argline` in the properties, i.e.:

```
	<properties>
		<tycho-surefire-argline>-Xms40m -Xmx1G -Dorg.testeditor.ignoreConnectionState=true</tycho-surefire-argline>
	</properties>
```